package/MDAnalysis/analysis/__init__.py

```diff
@@ -27,6 +27,20 @@
 
 The :mod:`MDAnalysis.analysis` sub-package contains various recipes and
 algorithms that can be used to analyze MD trajectories.
+If not stated differently, an analysis conducted by the available modules
+always follows the same structure
+
+1. Initialize the object previously imported.
+2. Run the analysis for specific trajectory slices
+3. Access the analysis from the `results` attribute (if available)
+
+ .. code-block:: python
+
+    from MDAnalysis.anlaysis import AnalysisModule
+
+    analysis_obj = AnalysisModule(<trajectory>, ...)
+    analysis_obj.run(<start>, <stop>, <step>)
+    print(analysis_obj.results)
```

Comment:
We typically do not use "placeholder" syntax but just write descriptive variable names.

Also, is this text actually rendered anywhere in the docs?

Reply:
Actually no. At least I didn't found this.